### ISSUE TITLE
Run ci periodically to detect changes after index entries have been merged.

### ISSUE THREAD

**Issue by mohammedzee1000** [CentOS/container-index#77]
I have noticed that a number of times, entries get invalidated, after the ci has passed and pr has been merged. Sometimes, a repo might get moved or a branch might get deleted or some scripts might get deleted or removed without updating index according. 

Hence i propose a periodic running of the ci script so that such invalidations can be caught. Also, i would recommend running the ci script as part of index reader so as to prevent builds from happening in case of entries getting invalidated prematurely.

**Comment by bamachrn** [comment]
I agree with the idea. In this case we need to plan for the next action also what we want to do with the invalid entries.

**Comment by mohammedzee1000** [comment]
https://github.com/CentOS/container-pipeline-service/pull/100 should allow us to run the testindex in cccp_reader as it now returns a list of failed entries (along with filenames) along with the pass or fail flag.

**Comment by mohammedzee1000** [comment]
We should probably add another ci job, that runs the index ci periodically, and sends out email to the team in case of failure in the index.

**Comment by navidshaikh** [comment]
>We should probably add another ci job, that runs the index ci periodically, and sends out email to the team in case of failure in the index.

Lets scope the issue (and its resolution) till detection of the invalid entries (just print them via a data structure in CI stdout). WDYT about CI job running once per week?

Actions per invalid entry can be scoped and discussed in a separate issue.

**Comment by bamachrn** [comment]
We need a sanity checking CI job for this.
